app/src/main/java/de/danoeh/antennapod/view/viewholder/EpisodeItemViewHolder.java

```diff
@@ -105,8 +106,14 @@ public void bind(FeedItem item) {
         isInQueue.setVisibility(item.isTagged(FeedItem.TAG_QUEUE) ? View.VISIBLE : View.GONE);
         container.setAlpha(item.isPlayed() ? 0.5f : 1.0f);
 
-        ItemActionButton actionButton = ItemActionButton.forItem(item, true, true);
-        actionButton.configure(secondaryActionButton, secondaryActionIcon, activity);
```

Comment:
Looks like this should be handled inside `ItemActionButton.forItem` instead

Reply:
yes, opened yet another PR :D